### PR TITLE
Add logo to document

### PR DESCRIPTION
**Pull request by hyrodium**
See more info at #63.

### CODE REVIEW

File: docs/logo.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    p_purple = Point(99,55)
+    p_1 = Point(13,96)
+    p_2 = Point(52,50)
+    p_3 = Point(73,73)
+    p_4 = Point(84,62)
+    p_5 = Point(116,97)
+
+    # Define path
+    path_dark = "docs/src/assets/logo-dark.png"
+    path_light = "docs/src/assets/logo.png"
```
Comment (johnnychen94) [inline docs/logo.jl:27]: The `docs/make.jl` file might be called from two typical paths:

- `project_dir`: `include("docs/make.jl")`
- `project_dir/docs`: `include("make.jl")`

The current version only handles the first case. To also support the second case, might be good to use `@__DIR__` macro here, which will be automatically expanded to `@__DIR__() = "project_dir/docs"`

Also use `joinpath` for windows users.

```suggestion
    path_dark = joinpath(@__DIR__, "src", "assets", "logo-dark.png")
    path_light = joinpath(@__DIR__, "src", "assets", "logo.png")
```

File: docs/logo.jl [hunk 2]
```diff
@@ -0,0 +1,47 @@
+function generate_logo()
+    # Load original image (from https://github.com/JuliaImages/juliaimages.github.io/blob/source/docs/src/assets/logo.png)
+    img = load("docs/logo_original.png")
```
Comment (johnnychen94) [inline docs/logo.jl:3]:
```suggestion
    img = load(joinpath(@__DIR__, "logo_original.png"))
````

**Review by johnnychen94 — APPROVED** [review]
Once the path issue get fixed, I think it's good to merge

File: docs/logo.jl (hunk 1 above)
Comment (hyrodium) [reply docs/logo.jl]: Thanks for the suggestion!

**Pull request merged**